examples/sketches/nearshore-test.geojson.json

```diff
@@ -78,9 +84,10 @@
                 "valueLabel": [
                     "Santa Maria"
                 ],
+                "formElementId": 2990,
                 "alternateLanguages": {
                     "pt": {
-                        "label": null,
+                        "label": "Ilha",
```

Comment:
I updated the sketch to have the latest attribute metadata including translations.  This was probably an old sketch example before we setup the translations